src/Toolkit/PubSub/V1/PublisherApi.php

```diff
@@ -0,0 +1,673 @@
+<?php
+/*
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+/*
+ * GENERATED CODE WARNING
+ * This file was generated from the file
+ * https://github.com/google/googleapis/blob/master/google/pubsub/v1/pubsub.proto
+ * and updates to that file get reflected here through a refresh process.
+ */
+
+namespace Google\Cloud\Toolkit\PubSub\V1;
+
+use Google\GAX\AgentHeaderDescriptor;
+use Google\GAX\ApiCallable;
+use Google\GAX\CallSettings;
+use Google\GAX\GrpcBootstrap;
+use Google\GAX\GrpcConstants;
+use Google\GAX\PageStreamingDescriptor;
+use Google\GAX\PathTemplate;
+use google\pubsub\v1\DeleteTopicRequest;
+use google\pubsub\v1\GetTopicRequest;
+use google\pubsub\v1\ListTopicSubscriptionsRequest;
+use google\pubsub\v1\ListTopicsRequest;
+use google\pubsub\v1\PublishRequest;
+use google\pubsub\v1\PublisherClient;
+use google\pubsub\v1\PubsubMessage;
+use google\pubsub\v1\Topic;
+
+/**
+ * Service Description: The service that an application uses to manipulate topics, and to send
+ * messages to a topic.
+ *
+ * This class provides the ability to make remote calls to the backing service through method
+ * calls that map to API methods. Sample code to get started:
+ *
+ * <pre>
+ * <code>
```

Comment:
Would we be able to use ``` instead of &lt;code&gt; in example snippets? It would be nice to keep consistency with the snippets we have in the veneer code.